drivers/SmartThings/matter-switch/src/init.lua

```diff
@@ -32,13 +33,36 @@ local COLOR_TEMPERATURE_MIRED_MIN = CONVERSION_CONSTANT/COLOR_TEMPERATURE_KELVIN
 
 local SWITCH_INITIALIZED = "__switch_intialized"
 local COMPONENT_TO_ENDPOINT_MAP = "__component_to_endpoint_map"
+local BOUNDS_CHECKED = "__bounds_checked"
+local COLOR_TEMP_BOUND_RECEIVED = "colorTemp_bound_received"
```

Comment:
Double underscore needed (idk the policy)?

Reply:
Updated to use double underscores in 775564f06809491f7f3c5359f887bf7864edad48.